tests/transport/spdp/spdp_transport.cpp

```diff
@@ -357,10 +356,27 @@ bool run_test()
   nonEmptyList[0].address[14] = 0;
   nonEmptyList[0].address[15] = 1;
 
-  OpenDDS::DCPS::LocatorSeq unicastLocators(2);
-  unicastLocators.length(2);
-  get_local(unicastLocators[0]);
-  unicastLocators[1] = nonEmptyList[0];
+  size_t addr_count;
+  ACE_INET_Addr* addr_array = 0;
+  const int ret = ACE::get_ip_interfaces(addr_count, addr_array);
+  struct Addr_Deleter {
```

Comment:
`ACE_Auto_Basic_Array_Ptr` does the same as Addr_Deleter, with C++11 we are using std::unique_ptr as alternative